doc/guide/scripting.rst

```diff
@@ -6,16 +6,49 @@
 Scripting Interface
 *******************
 
-Need some basic details here of how to load models and data via script, evaluate
-them at given parameter values and run bumps fits.
+Preparing your environment
+==========================
 
-The key functions are :func:`.core.load_model` for loading the
+You can use python scripts to load and plot your data, create SAS models and fit parameters. You can save a script to a file such as `example/model.py` and run
+it later. However, this requires a scripting environment with the correct packages installed.
+
+You can *either* use the SasView application itself (versions after 5.0.5), as
+both bumps and sasmodels are included as part of the distribution, so for
+example on Windows::
+
+    > sasview model.py
+ 
+(Note that it may be necessary to first add the folder path to sasmodels/sasview
+to your *Path* environment variable for this to work; set PATH=%PATH%;C:\\your\\path\\here\\)
+
+*or* create a Python environment with pip::
+
+    > pip install sasmodels sasdata matplotlib bumps periodictable
+    > python model.py
+
+(You can also create a Python environment using conda, see:
+https://github.com/SasView/sasview/wiki/DevNotes_CondaDevEnviroment)
+
+The pip command also works within a `Jupyter notebook <https://docs.jupyter.org/en/latest/install.html>`_ ::
+
+    %pip install sasmodels sasdata matplotlib bumps periodictable
+
+On a Mac the command for invoking SasView will be something like::
+
+    /Applications/Sasview.app/Contents/MacOS/sasview
+
+This can either be used directly or can be symlinked into your path::
+
+    mkdir ~/bin
+    ln -s /path/to/SasView.app/Contents/MacOS/sasview ~/bin
```

Comment:
I should have used the same string in both places, and the string should match the name in the .dmg. So `/Applications/SasView5.app`? I don't have my mac handy to check.

Do we need the caveat, "or whereever it is installed."?

Reply:
And I don't use a Mac... :-) How's this:
![image](https://user-images.githubusercontent.com/10938679/198634064-b43a6b11-1299-4af3-b99d-b513b5c1a222.png)